include/SFML/Graphics/Color.inl

```diff
@@ -147,3 +147,17 @@ constexpr Color& operator *=(Color& left, const Color& right)
 {
     return left = left * right;
 }
+
+
+////////////////////////////////////////////////////////////
+// Static member data
+////////////////////////////////////////////////////////////
+inline constexpr Color Color::Black(0, 0, 0);
+inline constexpr Color Color::White(255, 255, 255);
+inline constexpr Color Color::Red(255, 0, 0);
+inline constexpr Color Color::Green(0, 255, 0);
+inline constexpr Color Color::Blue(0, 0, 255);
+inline constexpr Color Color::Yellow(255, 255, 0);
+inline constexpr Color Color::Magenta(255, 0, 255);
+inline constexpr Color Color::Cyan(0, 255, 255);
+inline constexpr Color Color::Transparent(0, 0, 0, 0);
```

Comment:
`constexpr` implies `inline`, so it should work without that keyword.

Reply:
I feel like we're going in circles 😄 

https://github.com/SFML/SFML/pull/1904#discussion_r769082066
https://github.com/SFML/SFML/pull/1910#issuecomment-997181070

Reply:
> `constexpr` implies `inline`, so it should work without that keyword.

Without inline, VS2019 fails to compile as well, complaining about multiple definitions.

Reply:
> Without inline, VS2019 fails to compile as well, complaining about multiple definitions. My first commit did not use inline :)

I see, thanks. Maybe we should add a comment, otherwise someone in the future might wonder or even attempt to remove `inline`:
```cpp
// Note: inline keyword needed because ...
```